src/include/cpu/rpp_cpu_simd.hpp

```diff
@@ -2230,6 +2348,18 @@ inline void rpp_generic_nn_load_i8pln1(Rpp8s *srcPtrChannel, Rpp32s *srcLoc, Rpp
     p = _mm_unpacklo_epi8(px[0], px[1]);    // unpack to obtain [R01|R11|R21|R31|00|00|00|00|00|00|00|00|00|00|00|00]
 }
 
+inline void rpp_generic_nn_load_i8pln1_avx(Rpp8s *srcPtrChannel, Rpp32s *srcLoc, Rpp32s *invalidLoad, __m256i &p)
+{
+    Rpp8s buffer[16] = {0};
+    for(int i = 0; i < 8; i++)
+    {
```

Comment:
move the bracket to previous line. please follow google coding style

Reply:
Hi Rajy,
Currently all RPP functions follow the old RPP style where { will be in new line for functions
We planned to issue one global PR which will change all the instances to google style
Please let me know if you want this change to be done for water augmentation alone for now, and issue a global PR to change the style for the rest of the augmentations